terraform_project/main.tf

```
@@ -1,6 +1,6 @@
 terraform {
   backend "gcs" {
-    bucket  = "terraform-gantim"
+    bucket  = "nec-gcs-gnt-dev2"
```

```suggestion
    bucket  = "nec-gcs-gnt-dev"
```
If you want to do that in it's bucket doing this.